client-memory.html

```diff
@@ -108,23 +153,21 @@
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
-  
-<div class="section" id="omero-importer-omero-insight-on-unix-linux">
 
 <h2><a class="anchor" id="linux"></a>OMERO.insight on Unix/Linux</h2>
 
-<p>Changing OMERO.insight memory allocation under Unix requires you to edit the start-up script located with the clients, the <span class="bold">OMEROinsight_unix.sh</span> file.</p>
+<p>Changing OMERO.insight memory allocation under Unix requires you to edit the start-up script file, <span class="code-example">OMEROinsight_unix.sh</span> located with the clients.</p>
 
-<p>For the OMERO.insight GUI, you will need to edit the <span class="bold">OMEROinsight_unix.sh</span> file  and change the <span class="code">java</span> <span class="code">-Xms256000000</span> <span class="code">-Xmx1024000000</span> <span class="code">-jar</span> <span class="code">omero.insight.jar</span> line with new minimum and preferred memory options (see the <a class="reference external" href="http://docs.oracle.com/javase/6/docs/technotes/tools/solaris/java.html">Java online documentation</a> for a full description of these options).</p>
+<p>In the <span class="code-example">OMEROinsight_unix.sh</span> file, edit the <span class="code-example">java -Xms256000000 -Xmx1024000000 -jar omero.insight.jar</span> line with new minimum and preferred memory options.</p>
```

Comment:
This could maybe benefit from an extra line of explanation - you only talk about editing -Xmx above, doesn't suddenly mentioning -Xms here too confuse things? Do they need to edit both in this case?